core/components/archivist/elements/snippets/snippet.getarchives.php

```diff
@@ -49,7 +49,7 @@
 }
 
 /* get filter by year, month, and/or day. Sanitize to prevent injection. */
-$where = $modx->getOption('where',$scriptProperties,false);
+$where = $modx->getOption('where',$scriptProperties,"[]");
 $where = is_array($where) ? $where : $modx->fromJSON($where);
 $parameters = $modx->request->getParameters();
```

Comment:
If where contains an invalid JSON, the following line has to be added.

`$where = !empty($where) ? $where : array();`